src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,202 @@
+package duke;
+
+
+import duke.task.*;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+
+    //Application Logo
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    //Utilities
+    private static final String INDENTATION = "\t";
+    private static final int HORIZONTAL_LINE_LENGTH = 100;
+
+    //General Messages
+    private static final String MESSAGE_START_APPLICATION = "Hello from\n" + LOGO + System.lineSeparator() + "What can I do for you?";
+    private static final String MESSAGE_EXIT_APPLICATION = "Thank you for using our application. We hope to see you again soon";
```

Comment:
Note that some lines are >120 chars long, which deviates from the [Java coding standard layout](https://se-education.org/guides/conventions/java/basic.html#layout) guidelines. How about spreading out the message across the two lines?